src/main/java/com/quirko/gui/ScoreBoardPanel.java

```diff
@@ -0,0 +1,77 @@
+package com.quirko.gui;
+
+import java.awt.Color;
+import java.awt.event.*;
+import javax.swing.*;
+import javax.swing.border.EmptyBorder;
+import javax.swing.table.DefaultTableModel;
+import javax.swing.table.TableModel;
+
+public class ScoreBoardPanel extends JFrame implements KeyListener{
+
+    /*
+    It is the class that creates a window showing the scores.
+    */
+    private static final long serialVersionUID = 1L;
+    private JPanel contentPane;
+    private JTable table;
+    private String [] columns;
+    private String [][] rows;
+
+    public ScoreBoardPanel() {
+        this.setTitle("ScoreBoard");
+        this.setDefaultCloseOperation(DISPOSE_ON_CLOSE);
+        this.setBounds(400, 510, 450, 300);
+        this.setVisible(true);
+        this.setBackground(Color.DARK_GRAY);
+        this.addKeyListener(this);
+        this.setLocationRelativeTo(null);
+        this.setFocusable(true);
+
+        contentPane = new JPanel();
+        contentPane.setBorder(new EmptyBorder(5, 5, 5, 5));
+        setContentPane(contentPane);
+        contentPane.setLayout(null);
+        
+        JScrollPane scrollPane = new JScrollPane();
+        scrollPane.setBounds(28, 28, 396, 173);
+        contentPane.add(scrollPane);
+        
+        table = new JTable();
+        scrollPane.setViewportView(table);
+        
+        columns =new String[2];
```

Comment:
Score için de başka bir issue tanımlı. Şimdilik böyle kalabilir ama burası da burakinden daha kapsamlı şekilde implement edilmesi gerekiyor. 

Daha sonra o implementasyon buraya entegre olacak.